envs/example/defaults-2.0.yml

```diff
@@ -194,6 +194,9 @@ ceph:
   #  - sdd
   #bcache_ssd_device: sdg
 
+lvm:
+  enabled: false
```

Comment:
So this PR was all about enabling lvm for ci-full-* envs.. where in envs are we enabling lvm?